core/templates/components/skill-selector/select-skill-modal.component.ts

```diff
@@ -21,18 +21,22 @@ import { ConfirmOrCancelModal } from 'components/common-layout-directives/common
 import { ShortSkillSummary } from 'domain/skill/short-skill-summary.model';
 import { SkillSummaryBackendDict } from 'domain/skill/skill-summary.model';
 import { SkillsCategorizedByTopics } from 'pages/topics-and-skills-dashboard-page/skills-list/skills-list.component';
-
+export interface CategorizedSkills {
+  [topic: string]: {
+    [subtopic: string]: SkillSummaryBackendDict[];
+  };
+}
```

Comment:
Isn't this already defined somewhere?

Reply:
Oh yes, that's true. I came in when I was resolving conflicts between my branch #15049
I've just removed the changes